syntax = "proto3";
import "google/protobuf/timestamp.proto";

message Asset {
    int32 sid = 1;
    string symbol = 2;
    string asset_name = 3;
    google.protobuf.Timestamp start_date = 4;
    google.protobuf.Timestamp end_date = 5;
    google.protobuf.Timestamp first_traded = 6;
    google.protobuf.Timestamp auto_close_date = 7;
    string exchange = 8;
    string exchange_full = 9;
}